src/ui/public/chrome/api/theme.js

```diff
@@ -60,4 +60,20 @@ module.exports = function (chrome, internals) {
     return internals.brand[item];
   };
 
+  /**
+   * @param {string} val - the name of the theme
+   * @return {chrome}
+   */
+  chrome.setTheme = function (val) {
```

Comment:
I'm really not keen on exposing a `chrome.setTheme()` method which simply sets a `theme-xxx` class in the view.

Perhaps we could do `.addClass()` and `.removeClass()`, which the dashboard app can use to apply certain styles and anyone else can use for whatever purpose fits their use case.